Framework/Algorithms/test/FindPeakBackgroundTest.h

```diff
@@ -67,31 +67,14 @@ class FindPeakBackgroundTest : public CxxTest::TestSuite {
   /** Generate a workspace for test
    */
   MatrixWorkspace_sptr generateTestWorkspace() {
-    vector<double> data;
-    data.push_back(1);
-    data.push_back(2);
-    data.push_back(1);
-    data.push_back(1);
-    data.push_back(9);
-    data.push_back(11);
-    data.push_back(13);
-    data.push_back(20);
-    data.push_back(24);
-    data.push_back(32);
-    data.push_back(28);
-    data.push_back(48);
-    data.push_back(42);
-    data.push_back(77);
-    data.push_back(67);
-    data.push_back(33);
-    data.push_back(27);
-    data.push_back(20);
-    data.push_back(9);
-    data.push_back(2);
+
+    const size_t size = 20;
```

Comment:
This is a test with fixed inputs so `size = 20` shouldn't be a problem, but I'd tend to prefer getting it from the array after it is declared: `const size_t size = data.size()`. Especially in the next changes, where the number of items is large and it's easy to make a mistake.